src/back/Project.js

```diff
@@ -50,8 +50,19 @@ Project.prototype.render = function (force) {
     if (this.xml && !force) return this.xml;
     this.load(force);
     var renderer, Renderer;
-    if (this.mml) Renderer = require('./renderer/Carto.js').Carto;
-    else throw 'Oops, unkown renderer';
+    if (this.mml) {
```

Comment:
Nitpicking: not a big fan of using `switch` for a mapping, I'd suggest something like that instead:
```
MAP = {
    carto: require('./renderer/Carto.js').Carto,
    magnacarto: require('./renderer/Magnacarto.js').Magnacarto
};
if (!MAP[this.config.parsed_opts.renderer] || !this.mml) throw 'Unknown renderer';
Renderer = MAP[this.config.parsed_opts.renderer];
```